TypeScript.YAML-tmLanguage

```diff
@@ -169,7 +160,7 @@ repository:
       '5': { name: keyword.operator.ts }
       '6': { name: entity.name.function.ts }
       '7': { name: keyword.operator.ts }
-    end: '(?=\}|;|,|$)|(?<=\})'
```

Comment:
Sorry, by fine I meant your changes should be fine, not the code as it was originally written.